### PR TITLE
Translate KimuraSotaro_1.md

### LINKED ISSUE #50: KimuraSotaro_1 英訳
ref #49

### PR DESCRIPTION
**Pull request by eed3si9n**
Fixes #50 
/review @smroy

### CODE REVIEW

File: _candidates_en/KimuraSotaro_1.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+title: "Stream data processing 101"
+length: 40
+audience: Intermediate
+language: Japanese
+twitter: kimutansk
+github: kimutansk
+tags:
+  - Big Data / Fast Data
+---
+There's an increasing demand for stream data processing, which processes ever-occurring data in real-time, visualizes the results, or handles the resulting event. In this session, we will see some of the aspects of stream data processing, how they tackle the issues of batch processing, and it newly enables. Just in the last few years there have been a number of stream data processing platforms such as Spark Streaming, Flink, Apex, Gearpump, and Beam; so much so that it's unclear from the users' perspective where we are headed. By illustrating the history of stream data processing on the JVM, along with comparison of products that came out in the last few years, I hope to clarify the key points of the platforms that developers should know when building a stream data processing system.
```
Comment (smroy) [inline _candidates_en/KimuraSotaro_1.md]: Line 2: Replace "see" with "explore"?
Line 2: Who is "they"; developers? Need to define.
Line 3: Should "and it newly enables" read "and what technology it newly enables"?
Line 4: There should be a verb between "processing platform" and "such as"; perhaps add "created"?
Line 4: Replace "a number" with "many"?
Line 5: Should JVM be written out or is this a common abbreviation?
Line 5: Replace "know" with "understand"?

**Review by smroy — CHANGES_REQUESTED** [review]
Line 2: Replace "see" with "explore"?
Line 2: Who is "they"; developers? Need to define.
Line 3: Should "and it newly enables" read "and what technology it newly enables"?
Line 4: There should be a verb between "processing platform" and "such as"; perhaps add "created"?
Line 4: Replace "a number" with "many"?
Line 5: Should JVM be written out or is this a common abbreviation?
Line 5: Replace "know" with "understand"?

File: _candidates_en/KimuraSotaro_1.md (hunk 1 above)
Comment (eed3si9n) [reply _candidates_en/KimuraSotaro_1.md]: > Line 2: Replace "see" with "explore"?

Yes. _explore_ sounds better.

> Line 2: Who is "they"; developers? Need to define.

How about "how it tackles the issues unsolved by batch processing," where it = stream data processing?

> Line 3: Should "and it newly enables" read "and what technology it newly enables"?

Yes. That sounds much better.

> Line 4: There should be a verb between "processing platform" and "such as"; perhaps add "created"?
> Line 4: Replace "a number" with "many"?

How about "Many stream data processing platforms have emerged in the recent years, for instance Spark Streaming, Flink, Apex, Gearpump, and Beam;"?

> Line 5: Should JVM be written out or is this a common abbreviation?

For ScalaMatsuri JVM is common.

> Line 5: Replace "know" with "understand"?

Yes.

Comment (smroy) [reply _candidates_en/KimuraSotaro_1.md]: I've made all of these changes.

**Comment by eed3si9n** [comment]
When it looks all good, could you go to the "Files changed" tab and pick Approve under "Review changes"?

**Review by smroy — APPROVED** [review]

**Comment by smroy** [comment]
Done.

**Comment by eed3si9n** [comment]
Awesome. This gets reflected our website like this - http://2017.scalamatsuri.org/en/candidates/

**Comment by smroy** [comment]
That's really cool that it goes live right away!

**Pull request merged**